feat(util/template): add prettyVersion field (#16831)
renovatebot/renovate: docs/usage/templates.md

Returns `true` if at least one expression is `true`.

`{{#if (or isPatch isSingleVersion}}Small update, safer to merge and release.{{else}}Check out the changelog for all versions before merging!{{/if}}`

## prettifyVersion

> Appends `v` to start of `newMajor` or `newVersion`.

Reviewer: ```suggestion
Prepends `v` to start of `newMajor` or `newVersion` or any string not starting with `v`.
```

> If you want for example to create a pr title having the changed version:

Reviewer 2: ```suggestion
For example, to create a PR title with the changed version:
```

How about this?
